src/ngraph/env_util.hpp

```diff
@@ -18,28 +18,117 @@
 
 #include <string>
 
+#include "ngraph/ngraph_visibility.hpp"
+#include "ngraph/util.hpp"
+
+using namespace std;
+
 namespace ngraph
 {
+    /// \brief List of currently supported environment variables
+    enum class EnvVarEnum : uint32_t
```

Comment:
This technique does not allow for other backends to add env vars. If we are to have a solution it should work for backends as well as ngraph